include/zephyr/sensing/sensing.h

```diff
@@ -110,7 +115,8 @@ typedef void *sensing_sensor_handle_t;
  */
 typedef void (*sensing_data_event_t)(
 		sensing_sensor_handle_t handle,
-		const void *buf);
+		const void *buf,
+		void *context);
```

Comment:
needs doxygen comment

Reply:
Thanks. Submitted https://github.com/zephyrproject-rtos/zephyr/pull/68317 to fix it.